src/dcat-us/dataset-formatter.ts

```diff
@@ -1,122 +1,35 @@
 
-// TODO - use real type for hubDataset when it gets defined in Hub.js
-type HubDatasetAttributes = Record<string, any>;
 export type DcatDatasetTemplate = Record<string, any>;
 
-export function formatDcatDataset (hubDataset: HubDatasetAttributes, siteUrl: string, siteModel: IModel, datasetTemplate: DcatDatasetTemplate) {
-  // Download and Hub Links must be generated from Content
-  const content = datasetToContent({ 
-    id: hubDataset.id, 
-    attributes: hubDataset
-  } as DatasetResource);
-  const { relative: relativePath } = getContentSiteUrls(content, siteModel);
-  const hubLandingPage = siteUrl.startsWith('https://') ? siteUrl + relativePath : `https://${siteUrl}${relativePath}`;
-  const downloadLink = siteUrl.startsWith('https://') ? `${siteUrl}/datasets/${content.identifier}` : `https://${siteUrl}/datasets/${content.identifier}`;
-
-  // AGO links must be generated from Dataset Records
-  const { itemId, layerId } = parseDatasetId(hubDataset.id);
-  let agoLandingPage = `${portalUrl}/home/item.html?id=${itemId}`;
-  if (layerId) {
-    agoLandingPage += `&sublayer=${layerId}`;
-  }
-
-  const { 
-    structuredLicense: { url = null } = {},
-    licenseInfo = ''
-  } = hubDataset;
-
-  // Override hub.js default license value of 'none'
-  if (hubDataset.license === 'none') {
-    hubDataset.license = null;
-  }
+export function formatDcatDataset(hubDataset: any, datasetTemplate: DcatDatasetTemplate, transforms: TransformsList, nonEditableFieldPaths: string[]) {
 
   const defaultDataset = {
-    '@type': 'dcat:Dataset',
-    identifier: agoLandingPage,
-    landingPage: hubLandingPage
-  };
-
-  const transforms: TransformsList = {
-    toISO (_key, val) {
-      return new Date(val).toISOString();
-    },
-    toArray (_key, val) {
-      if (!val) return [];
-      else return _.castArray(val);
-    }
+    '@type': 'dcat:Dataset'
   };
 
   const dcatDataset = Object.assign({}, defaultDataset, adlib(datasetTemplate, hubDataset, transforms));
 
   // reset uninterpolated, non-editable fields since customers cannot remove them
   resetUninterpolatedPaths(dcatDataset, nonEditableFieldPaths);
-
-  if (!dcatDataset.license || dcatDataset.license.match(/{{.+}}/g)?.length) {
-    dcatDataset.license = url || licenseInfo || '';
-  }
-
-  if (isPage(hubDataset as IItem) && !hasTags(hubDataset)) {
-    dcatDataset.keyword = ['ArcGIS Hub page'];
-  }
-
-  dcatDataset.distribution = _generateDistributions({
-    hubDataset, 
-    dcatDataset,
-    landingPage: hubLandingPage, 
-    downloadLink
-  });
-
-  const spatial = computeSpatialProperty(datasetTemplate, dcatDataset);
-  if (spatial) {
-    dcatDataset.spatial = spatial;
-
-    // https://project-open-data.cio.gov/v1.1/schema/#theme
-    // allow theme to be overridden
-    if (_.isEmpty(datasetTemplate.theme)) {
-      dcatDataset.theme = ['geospatial'];
-    }
-  } else if (dcatDataset.spatial) {
-    delete dcatDataset.spatial;
+  if (dcatDataset.distribution) {
```

Comment:
what is a distribution and where does it come from?

Reply:
@rgwozdz Distribution is one of the key in DCAT US format that contains an array of an object containing download url/access url of the dataset in various formats. Users can add distribution via templates.